uai2017/python/create-invitations.py

```diff
@@ -23,8 +23,9 @@ def get_open_comment_invitation(submissionId, number, authorsGroupId):
     allGroups = [COCHAIRS, PC, SPC, authorsGroupId]
     reply = {
         'forum': submissionId,
+        'selfReplyOnly': True,
         'signatures': {
-            'values-regex': '|'.join(allGroups),
+            'values-regex': '|'.join(allGroups) + '|' + CONFERENCE + '/Paper' + str(number) + '/AnonReviewer[0-9]+',
```

Comment:
I would move CONFERENCE + '/Paper' + str(number) + '/AnonReviewer[0-9]+' to the allGroups list, just to keep things consistent... what do you think?